packages/polling/src/poll.ts

```diff
@@ -395,7 +422,7 @@ export namespace Poll {
     standby?: Standby | (() => boolean | Standby);
   }
   /**
-   * An interval value (0ms) that indicates the poll should tick immediately.
+   * An interval value that indicates the poll should tick immediately.
```

Comment:
Probably worth mentioning the unit of the value.

```suggestion
   * An interval value in ms that indicates the poll should tick immediately.
```